frontend/src/employee-mobile-frontend/components/staff/StaffPage.tsx

```diff
@@ -78,15 +80,15 @@ export default React.memo(function StaffPage() {
   const today = useMemo(() => LocalDate.today(), [])
 
   return (
-    <>
+    <FixedSpaceColumn spacing="zero" style={{ height: '100vh' }}>
```

Comment:
These layout changes make this tab behave inconsistently with the other tabs. The top bar seems to be squeezed a little bit.

<img width="508" alt="image" src="https://user-images.githubusercontent.com/1176169/148739320-ce4399d3-2d73-49b4-9e6b-0fe1337139f7.png">

Either revert the layout change or make all tabs behave similarly. This could also be done in a separate PR.

Reply:
Fixed this by adding a `min-height` rule to the top bar

Reply:
Well, that's just one instance affected by the squeezing problem 😅 

<img width="516" alt="image" src="https://user-images.githubusercontent.com/1176169/148743593-d5ab1a76-1851-4048-9240-1e0a6807fa7f.png">
<img width="458" alt="image" src="https://user-images.githubusercontent.com/1176169/148743627-df543ffe-cdb1-44bd-9592-36faf99b658f.png">

I think all pages should be wrapped in the same container components, or we'll be always battling these small differences.

Reply:
Splitting this off to a separate PR